lib/puppet/type/mailalias.rb

```diff
@@ -33,6 +30,11 @@ def should_to_s(value)
       end
     end
 
+    newproperty(:file) do
```

Comment:
Was that a conclusion that we can leave it as-is for this PR and someone (probably me) can try switching them in the instances @adrienthebo identified, or that the PR should be changed to use `newparam`?